aws-ami-creation/scripts/install_huggingface_libraries.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+echo "Step: install-hugging-face-libraries"
+
+echo "Activating the virtual-env"
+source /opt/aws_neuron_venv_pytorch/bin/activate
```

Comment:
This would be a big breaking change to the existing AMI. Where you can straight up use "python". Can we either activate it by default or not use a virtual env?

Reply:
The base AMI let's us use the neuron package and other pre-installed packages only if we activate this environment. If we plan to not use a virtual env then we need to install everything just like the earlier repo.

Reply:
And in that case we should use https://aws.amazon.com/releasenotes/aws-deep-learning-ami-base-neuron-ubuntu-20-04/ as mentioned in the documentation that 

```
The Base AMI comes with a foundational platform of GPU drivers and acceleration libraries to deploy your own customized deep learning environment. 
```

Reply:
If those are only installed in `venv` then we should defaulty activate those on instance start and ssh connection.